### ISSUE TITLE
Centered UDButton Icons when no Text is specified

### ISSUE THREAD

**Issue by donlent** [ironmansoftware/powershell-universal#1045]
### Summary of the new feature / enhancement

Sometimes you'd like to have Buttons with just an Icon (I know you can use UDIconButton but they are not the same... ;-)
Especially when you're dealing with Responsive UIs and need to save some space.

In this example you can clearly see that the Icon is always slightly left-justified:
![image](https://user-images.githubusercontent.com/11975745/158116319-70fcc50c-85be-4937-affc-49b9ce6dc0bb.png)

It would be nice if, in case of absent Text, the Icon would be centered.

Also I noticed that for some reasons it's not possible to resize the button to smaller than the shown width in my example. You can go higher but not smaller. The size spec will be ignored somehow...

Thanks,
Don

**Comment by adamdriscoll** [comment]
I actually can't reproduce this. Can you share your code? I wonder if it's because it's in a table. 

```
New-UDButton -Icon (New-UDIcon -Icon trash) -Style @{ color = 'red';  } 
```

![image](https://user-images.githubusercontent.com/1256531/160139388-5500b0ea-522a-4f7e-936c-8a7f8241e0d6.png)

I was also able to change the button size with minWidth and padding. 

```
New-UDButton -Icon (New-UDIcon -Icon trash) -Style @{ color = 'red'; minWidth = '10px'; padding = '5px'  } 
```

![image](https://user-images.githubusercontent.com/1256531/160139688-ad392fa9-4c32-4934-8040-ac342bff539a.png)

**Comment by donlent** [comment]
Hi Adam
Very interesting. On my side its clearly reproducable just by using your code...
![image](https://user-images.githubusercontent.com/11975745/160330704-0a26c624-433c-4b4d-99d3-f69ffaf44ea8.png)

Could it be that this issue has been resolved in a post 2.5.5 release?
In any case I can try in the next few days using a newer build and let you know in case you don't remember if this issue had already been adressed...

Best regards,
Don

**Comment by donlent** [comment]
Dear Adam
I can confirm that using the current 2.9.3 Prod Release it works as expected! So it seems that you've already fixed that by yourself. ;-)
Thanks a lot for your efforts!
Best regards,
Don
